sample/0.1.0/content/entrypoint.ps1

```diff
@@ -0,0 +1,12 @@
+param(
+    [Alias('f')]
+    [Switch]$Fail
+)
+
+echo "This is a script used to test PRM"
+echo "Contents of target dir: "
+ls
```

Comment:
In PowerShell, when writing strings as output, it's best practice to simply emit the string; this is functionally the same as `echo` or `Write-Output`, but with less overhead. It is also idiomatic not to use aliases in scripts (and `ls` is an alias of `Get-ChildItem`).

```suggestion
"This is a script used to test PRM"
"Contents of target dir: "
Get-ChildItem
```